Javascript/index.js

```diff
@@ -47,4 +49,156 @@ const updateFormValues = () => {
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+];
+
+const btnsM = document.querySelectorAll('.buttonMobile');
+const btns = document.querySelectorAll('.buttons');
+const divProject = document.createElement('div');
+
+function addEventListenerList() {
+  for (let i = 0; i < btns.length; i += 1) {
+    btns[i].addEventListener('click', (event) => {
+      const projectId = event.target.id;
+      divProject.id = 'modal';
+      divProject.className = 'modal flex';
+      document.querySelector('.overal').style.display = 'flex';
+
+      let techHTML = '';
+      for (let l = 0; l < projects[i].tech.length; l += 1) {
+        techHTML += `<li class="modal-language">${projects[i].tech[l]}</li>`;
+      }
+
+      divProject.innerHTML = '<section class="modal-header flex">'
+        + '<div class="top-side-modal flex">'
```

Comment:
Good job on making the required changes, but there are some left to be done. A few more steps and it will be good, ai know you can do it.

- [ ] **Required:** The popup doesn't work on mobile version, make sure it works and is responsive. 
![image](https://user-images.githubusercontent.com/2903437/128616579-820d0e94-4609-4453-974b-cb3c1278c89d.png)